src/task.ts

```diff
@@ -327,15 +327,18 @@ export default class Task {
     name: string,
     address: string,
     constructorArguments: string | unknown[],
+    externalTask?: Task,
     libs?: Libraries
```

Comment:
This looks correct, and I think we don't have any real use case with libs so far for `verify` itself. The only case with libraries I see in the repository is managed pools; however they use `deployAndVerify` with libs, not just `verify` so it's not really affected by this change.

But given that this a very specific feature for this use case, I would be leaning towards placing the new argument at the end, ensuring that it's 'backwards compatible' no matter what.

```suggestion
    libs?: Libraries,
    externalTask?: Task
```